.github/ISSUE_TEMPLATE/refactor.yml

```diff
@@ -0,0 +1,11 @@
+name: Refactor (internal)
+description: For internal use only.
+labels: ['type:refactor', 'status:requirements', 'priority-5-triage']
+body:
+  - type: textarea
+    id: describe-the-proposed-refactor
+    attributes:
+      label: Describe the proposed refactor.
```

Comment:
Is `refactor` a verb? 

I think people use `refactoring` to mean: "the act of cleaning up existing code", so that would be a verb for sure. But I'm not sure if `refactor` on its own is a verb.

I'll let a native English speaker decide. Feel free to make a suggestion to change this. 😉

Reply:
Not a native speaker, but even though it is not in a dictionary, it is a verb for programmers